features/mathml.yml.dist

```diff
@@ -2,8 +2,9 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
+  baseline: high
```

Comment:
I checked BCD for open issues and PRs in MathML, none found. I also checked Chromium, WebKit, and Gecko issues, which was less helpful but I wasn't able to find anything especially interesting for this feature.